dev-python/safety/files/safety-3.5.2-0005-opt-in-telemetry.patch

```diff
@@ -0,0 +1,65 @@
+From 96820d5c85e53fad4952c6a1d4a6de261384f3bb Mon Sep 17 00:00:00 2001
+From: Oz Tiram <oz.tiram@gmail.com>
+Date: Sat, 5 Jul 2025 14:03:59 +0200
+Subject: [PATCH] make telemetry opt-in
```

Comment:
Is that an upstream change? Overall, I don't like the idea of having patches that change CLI, so it'd be best if you worked with upstream to get something that works consistently for us and them together.

Reply:
I will do this, but please don't let this block, upstream is really slow integrating changes that aren't coming from their team.

Reply:
PR upstream opened. See https://github.com/pyupio/safety/pull/774.